docs/user-guide/geo.rst

```diff
@@ -0,0 +1,70 @@
+Geospatial Data
+===============
+
+What geospatial data is supported?
+----------------------------------
+
+In general, the Python Geo Interface is supported; see https://gist.github.com/sgillies/2217756
+
+There are three layers of information:
+
+- The outermost layer is a FeatureCollection
+- Within a FeatureCollection are Features. Each feature has a type, an id, an optional
```

Comment:
In general, most people are not accustomed to working with geospatial data unless it is their particular profession, so I think even more details are appropriate.

Most of your bullets are short, but this one stands out as long. You may want to break this into more parts with expanded descriptions:
- Within a FeatureCollection are Featrues.
- A Feature is an object that represents XXX. It in turn contains:
    - type - examples or description
    - id - unique identifier for a given instance of a Feature object
    - geometry (optional) - shape?
    - bbox (optional) - the bounding box information about the geographic shape being represented in this Feature.
    - additional optional properties - see here <-- make this a link to specific docs